opentelemetry-instrumentation/src/opentelemetry/instrumentation/environment_variables.py

```diff
@@ -16,3 +16,13 @@
 """
 .. envvar:: OTEL_PYTHON_DISABLED_INSTRUMENTATIONS
 """
+
+OTEL_PYTHON_DISTRO = "OTEL_PYTHON_DISTRO"
```

Comment:
are these vars added to specification as well?

Reply:
> @jeremydvoss : can you please add some test cases for this change?

I would like to. Sitecustomize tests don't currently exist. I'm guessing this is because importing the module triggers autoinstrumentation. So I will need to figure out some workaround to enable testing...

Reply:
Actually, since this is python specific, a cross language spec update does not seem to make sense. But I can update the docs:
https://github.com/open-telemetry/opentelemetry-python-contrib/blob/a6797541721af72e6feab400e24ceaabc08cb31e/opentelemetry-instrumentation/README.rst#L93

https://github.com/open-telemetry/opentelemetry.io/blob/54d9c9e97a7f4afda78a90939835e71bf263db79/content/en/docs/instrumentation/python/automatic/agent-config.md?plain=1#L159

Reply:
https://github.com/open-telemetry/opentelemetry.io/pull/2788